crates/ide_assists/src/handlers/add_explicit_type.rs

```diff
@@ -180,14 +180,17 @@ fn main() {
 }
 "#,
         );
-        // note: this may break later if we add more consteval. it just needs to be something that our
-        // consteval engine doesn't understand
-        check_assist_not_applicable(
+        check_assist(
```

Comment:
this is supposed to test a case where the assist isn't applicable, so would you mind changing it to something else and leave the comment?

Reply:
Ah I missed the comment's point completely